### ISSUE TITLE
Task for this homework

### ISSUE THREAD

**Issue by pratibha77118** [ubsuny/23-Homework7G1#20]
Hi all,
What exactly are you going to do in this task? Even though we chose different configuration, we are supposed to perform the same task with different configuration. There is no task distribution for this homework, isn't it?

**Comment by reshnashrestha** [comment]
@pratibha77118 Yes, we just have to make sure we choose the different pair of configuration and perform the same task, and at the last we have to discuss our approach.
